client/App.vue

```diff
@@ -181,6 +190,15 @@ export default {
       <div class="detail-view task-queue" v-if="$route.params.taskQueue">
         <span>{{ $route.params.taskQueue }}</span>
       </div>
+      <div v-if="currentUser" class="user">
+        <img
+          :src="currentUser.picture"
+          width="200"
+          alt="user pic"
+          class="avatar"
+        />
+        <span class="name">{{ currentUser.name }}</span>
```

Comment:
```suggestion
        <a href="/signin"><span class="name">{{ currentUser.name }}</span></a>
```

can also put the a tag around the img, but then the CSS would need to change

Reply:
updated